torch/csrc/jit/passes/quantization.cpp

```diff
@@ -1826,13 +1828,16 @@ std::tuple<c10::QScheme, QParamVector> InsertQuantDeQuantHelper::
 
 c10::optional<Module> InsertQuantDeQuantHelper::findChildModuleToQuantize(
     Module& module,
-    Value* child_instance) {
+    Node* n,
+    Value* self) {
+  Value* child_instance = n->inputs()[0];
   TORCH_INTERNAL_ASSERT(
       child_instance->node()->kind() == prim::GetAttr,
       "Child instance should come from GetAttr.");
```

Comment:
this can be removed, we'll check this in getInvokedModule

Reply:
We dont currently check this there, are you suggesting we add it?